tests/functionsTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class createDBConnectionTest extends TestCase
+{
+    function testSuccessCreateDBConnection()
+    {
+
```

Comment:
This test appears to be empty!

You can't actually unit test the createDBConnection function, because it interacts with a DB the test would be testing both the function and the database. 
 
From your functions, the only one that can be unit tested is `createRecipeCards` so would suggest changing this test to that